CHANGELOG.rst

```diff
@@ -42,6 +42,10 @@ Added
   client commands for interacting with Inquiries
 
   Contributed by mierdin. #3653
+* Added two new rule operators, `inside` and `ninside` which allow for the reverse action of
```

Comment:
I would not use the word "action" here to avoid confusion with stackstorm's actions